src/libs/SearchUIUtils.ts

```diff
@@ -501,7 +501,10 @@ function getTransactionsSections(data: OnyxTypes.SearchResults['data'], metadata
         // Use Map.get() for faster lookups with default values
         const from = personalDetailsMap.get(transactionItem.accountID.toString()) ?? emptyPersonalDetails;
         const to = transactionItem.managerID && !shouldShowBlankTo ? (personalDetailsMap.get(transactionItem.managerID.toString()) ?? emptyPersonalDetails) : emptyPersonalDetails;
+        // const isPolicyExpenseChat = !!reports.find((rp) => rp.policyID === transactionItem.policyID && rp.isPolicyExpenseChat);
 
+        // console.log({reports})
+        // console.log({isPolicyExpenseChat});
```

Comment:
Why is all this code commented out? I see some more below here. We should not merge this kinda changes into `main`

Reply:
Thread here https://expensify.slack.com/archives/C08CZDJFJ77/p1751048080518369